astropy/units/format/generic.py

```diff
@@ -469,11 +469,14 @@ def parse(cls, s, debug=False):
             s = s.decode('ascii')
 
         result = cls._do_parse(s, debug=debug)
-        if s.count('/') > 1:
-            warnings.warn(
-                "'{}' contains multiple slashes, which is "
-                "discouraged by the FITS standard".format(s),
-                core.UnitsWarning)
+        nsolidus = s.count('/')
```

Comment:
Since `vounit` inherits from this class, how about a new (class) method `more_than_one_solidus`?